mods/d2k/rules/defaults.yaml

```diff
@@ -131,13 +123,16 @@
 	TargetableUnit:
 		TargetTypes: Ground
 		RequiresForceFire: yes
-	Capturable:
-		Type: husk
-		AllowAllies: yes
-		CaptureThreshold: 1.0
-	TransformOnCapture:
-		ForceHealthPercentage: 25
+#	Capturable:
+#		Type: husk
+#		AllowAllies: yes
+#		CaptureThreshold: 1.0
+#	TransformOnCapture:
+#		ForceHealthPercentage: 25
```

Comment:
I know that but I wasn't sure if people really want certain features removed. I do, but to save more work if there are complaints I left it in. If people are willing to go along with it then the comments can all be removed.